setup.py

```diff
@@ -295,7 +302,9 @@ def cythonize(x, **kwargs):
         'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
     ],
     packages=['primer3'],
-    ext_modules=[primerdesign_ext] + try_cythonize([thermoanalysis_ext]),
+    ext_modules=try_cythonize(cython_extensions),
+    # ext_modules=[primerdesign_ext] + try_cythonize([thermoanalysis_ext]),
+    # ext_modules=try_cythonize([thermoanalysis_ext]),
```

Comment:
Any reason to keep these two lines at this point?

Reply:
nope: deleted